src/dmd/func.d

```diff
@@ -3150,6 +3197,60 @@ private bool checkEscapingSiblings(FuncDeclaration f, FuncDeclaration outerFunc,
     return bAnyClosures;
 }
 
+/***
+ * Returns true if any of the symbols `p` resides in the enclosing instantiation scope of `s`.
+ */
+bool followInstantiationContext(D...)(Dsymbol s, D p)
```

Comment:
are you positif that that you need a variadic template here?

Reply:
No I don't. The way it happened is I only used one `p` (parent symbol) then later I realised that with inlining I needed to check a second one which is the parent of the original function where the inline symbol came from so I made it a template instead of an array because the former for sure would be unrolled and the rare case of a second parent isn't worth the loop time for the usual single `p` case. could be an overload with two `p`s but I like it this way.